### ISSUE TITLE
Should verse references in URLs but not in the post's plain text be indexed?

### ISSUE THREAD

**Issue by curiousdannii** [alerque/stack-verse-mapper#25]
For a great number of examples, see [this Hermeneutics.SE post](http://hermeneutics.stackexchange.com/a/22033/2757).

> All have the [spirit](https://www.biblegateway.com/passage/?search=Job+33%3A4&version=RSVCE), however what defines one as having the "Holy Spirit" its that [out of our mouth we speak only clean things](https://www.biblegateway.com/passage/?search=Matthew+15%3A11&version=RSVCE). This can only occur when the [water of Jesus's words](https://www.biblegateway.com/passage/?search=John+4%3A14&version=RSVCE) has been [consumed](https://www.biblegateway.com/passage/?search=John+7%3A37&version=RSVCE). For until the water gets consumed we [rejected the testimonies of what was seen and known](https://www.biblegateway.com/passage/?search=John+3%3A11&version=RSVCE). For those that [do what Jesus told them and listen](https://www.biblegateway.com/passage/?search=Matthew+11%3A15&version=RSVCE) will [hear and see what takes place because what was spoken](https://www.biblegateway.com/passage/?search=Mark+4%3A12&version=RSVCE) learning the difference between the [fresh water and the salt](https://www.biblegateway.com/passage/?search=James+3%3A11&version=RSVCE). Thereby not releasing the [speech of error](https://www.biblegateway.com/passage/?search=Matthew+12%3A43-45&version=RSVCE), and instead [binding it within ourselves](https://www.biblegateway.com/passage/?search=Matthew+12%3A29&version=RSVCE). Freeing us from the curse of the [discernment of Function and Error](https://www.biblegateway.com/passage/?search=Genesis+2%3A17&version=RSVCE).

My inclination is to say no, if a verse reference isn't directly mentioned in the plain text of a post then it isn't important enough to be indexed.

**Comment by alerque** [comment]
What about adding a level to our priority stack (verse references in headers score higher, verses inside ranges score lower than direct references, etc.) to give them less relevance but still include the signal somehow that says these verses are possibly related?

**Comment by curiousdannii** [comment]
Yeah that would be possible. It would require substantial changes to both the index structure and the search algorithm, so lets consider it again after the app is more functional.

**Comment by alerque** [comment]
Savvy.
